### PR TITLE
add-travel-mode

### PR DESCRIPTION
**Pull request by daothanhcam**

### CODE REVIEW

File: app/views/addresses/index.html.erb [hunk 1]
```diff
@@ -6,8 +6,20 @@
...
 <%= render "shared/button" %>
+<input id="pac-input" class="controls" type="text">
+
+<div id="mode">
+  <strong><%= t "directions.travel_mode" %></strong></br>
+  <select id="travel-mode" onchange="calcRoute();">
+    <option value="DRIVING"><%= t("directions.driving") %></option>
+    <option value="WALKING"><%= t("directions.walking") %></option>
+    <option value="BICYCLING"><%= t("directions.bicycling") %></option>
+    <option value="TRANSIT"><%= t("directions.transit") %></option>
```
Comment (samnangcattor) [inline app/views/addresses/index.html.erb]: delete ()  ,  t("directions.transit") => t"directions.transit"

File: app/assets/javascripts/googlemapapi.js [hunk 2]
```diff
@@ -27,6 +29,9 @@ function initialize() {
     title:"Property Location"
```
Comment (bapboy18) [inline app/assets/javascripts/googlemapapi.js]: indent

File: app/views/addresses/index.html.erb [hunk 3]
```diff
@@ -6,8 +6,20 @@
...
 <%= render "shared/search" %>
 <%= render "shared/button" %>
+<input id="pac-input" class="controls" type="text">
+
+<div id="mode">
+  <strong><%= t "directions.travel_mode" %></strong></br>
+  <select id="travel-mode" onchange="calcRoute();">
+    <option value="DRIVING"><%= t("directions.driving") %></option>
+    <option value="WALKING"><%= t("directions.walking") %></option>
+    <option value="BICYCLING"><%= t("directions.bicycling") %></option>
```
Comment (samnangcattor) [inline app/views/addresses/index.html.erb]: delete (), t("directions.bicycling") => t"directions.bicycling"

File: app/views/addresses/index.html.erb [hunk 4]
```diff
@@ -6,8 +6,20 @@
...
 <%= render "shared/current_location" %>
 <%= render "shared/search" %>
 <%= render "shared/button" %>
+<input id="pac-input" class="controls" type="text">
+
+<div id="mode">
+  <strong><%= t "directions.travel_mode" %></strong></br>
+  <select id="travel-mode" onchange="calcRoute();">
+    <option value="DRIVING"><%= t("directions.driving") %></option>
+    <option value="WALKING"><%= t("directions.walking") %></option>
```
Comment (samnangcattor) [inline app/views/addresses/index.html.erb]: delete (), t("directions.walking") => t"directions.walking"

File: app/views/addresses/index.html.erb [hunk 5]
```diff
@@ -6,8 +6,20 @@
...
 <div id="map-canvas"></div>
 <%= render "shared/current_location" %>
 <%= render "shared/search" %>
 <%= render "shared/button" %>
+<input id="pac-input" class="controls" type="text">
+
+<div id="mode">
+  <strong><%= t "directions.travel_mode" %></strong></br>
+  <select id="travel-mode" onchange="calcRoute();">
+    <option value="DRIVING"><%= t("directions.driving") %></option>
```
Comment (samnangcattor) [inline app/views/addresses/index.html.erb]: delete (), t("directions.driving") => t"directions.driving"

File: app/assets/javascripts/googlemapapi.js [hunk 6]
```diff
@@ -27,6 +29,9 @@ function initialize() {
...
   });
 
+  directionsDisplay = new google.maps.DirectionsRenderer();
+  directionsDisplay.setMap(map)
```
Comment (tranhuyhoangbka) [inline app/assets/javascripts/googlemapapi.js]: have no ";"

**Pull request merged**